src/libtools/property.cc

```diff
@@ -50,15 +52,15 @@ const Property &Property::get(const string &key) const {
   }
 
   const Property *p;
-  map<string, Index>::const_iterator iter;
+  map<string, std::vector<Index>>::const_iterator iter;
```

Comment:
There is now a vector of indices in order to represent multiple attributes, right?

Reply:
yes to have indeces to all the properties which have the same tag.